triage/summarize/files.go

```diff
@@ -143,14 +143,16 @@ prints a warning if the results could not be memoized.
 message is a message that gets printed on success, appended to "Done ". If it is the empty
 string, no message is printed.
 */
-func memoizeResults(filepath string, message string, v interface{}) {
+func memoizeResults(filepath, message string, v interface{}) {
 	err := writeJSON(filepath, v)
-	if err == nil && message != "" {
-		logInfo("Done " + message)
+	if err != nil {
+		logWarning("Could not memoize results to '%s': %s", filepath, err)
 		return
 	}
 
-	logWarning("Could not memoize results to '%s': %s", filepath, err)
+	if message != "" {
+		logInfo("Done " + message)
+	}
```

Comment:
This change will eventually need to use klog, so it should only be merged after #18858